Mathlib/RingTheory/SimpleModule.lean

```diff
@@ -119,6 +120,18 @@ theorem is_semisimple_iff_top_eq_sSup_simples :
     exact IsSemisimpleModule.sSup_simples_eq_top⟩
 #align is_semisimple_iff_top_eq_Sup_simples is_semisimple_iff_top_eq_sSup_simples
 
+lemma isSemisimpleModule_of_IsSemisimpleModule_submodule {s : Set ι} {p : ι → Submodule R M}
+    (hp : ∀ i ∈ s, IsSemisimpleModule R (p i)) (hp' : ⨆ i ∈ s, p i = ⊤) :
+    IsSemisimpleModule R M := by
```

Comment:
Could you please state versions of this lemma where the sup is not necessarily `\top`? Maybe just a lemma that is directly about the supremum.

Reply:
I have added the lemma `isSemisimpleModule_biSup_of_IsSemisimpleModule_submodule` below.